app/cdap/components/WrangleHome/Components/OngoingDataExplorations/index.tsx

```diff
@@ -14,102 +14,148 @@
  * the License.
  */
 
-import React, { useEffect, useState } from 'react';
 import { Box } from '@material-ui/core/';
-import { generateDataForExplorationCard } from './utils';
 import MyDataPrepApi from 'api/dataprep';
+import { getCategorizedConnections } from 'components/Connections/Browser/SidePanel/apiHelpers';
+import {
+  IConnectionWithConnectorType,
+  IExistingExplorationCardsData,
+  IConnectionsList,
+  IMassagedObject,
+  IValues,
+} from 'components/WrangleHome/Components/OngoingDataExplorations/types';
+import { getUpdatedExplorationCards } from 'components/WrangleHome/Components/OngoingDataExplorations/utils';
+import OngoingDataExplorationsCard from 'components/WrangleHome/Components/OngoingDataExplorationsCard';
+import T from 'i18n-react';
+import { orderBy } from 'lodash';
+import React, { useCallback, useEffect, useState } from 'react';
 import { Link } from 'react-router-dom';
-import { getCurrentNamespace } from 'services/NamespaceStore';
-import OngoingDataExplorationsCard from '../OngoingDataExplorationsCard';
-import { switchMap } from 'rxjs/operators';
 import { forkJoin } from 'rxjs/observable/forkJoin';
-import { IResponseData } from './types';
-import T from 'i18n-react';
+import { defaultIfEmpty, switchMap } from 'rxjs/operators';
+import { getCurrentNamespace } from 'services/NamespaceStore';
+
+export default function() {
+  const [onGoingExplorationsData, setOnGoingExplorationsData] = useState<IMassagedObject[]>([]);
+
+  const getOngoingData = useCallback(async () => {
+    const connectionsWithConnectorTypeData: Map<
+      string,
+      IConnectionsList[]
+    > = await getCategorizedConnections();
+    const connectionsWithConnectorTypeDataObject: IConnectionWithConnectorType[] = [];
+    for (const connectorName of connectionsWithConnectorTypeData.keys()) {
+      const values = connectionsWithConnectorTypeData.get(connectorName);
+      const connections = values.map((eachValue) => {
```

Comment:
arrow func without any logic and single return statement can be changed to shorthand ()=> statement

Reply:
done